src/System.Net.NetworkInformation/tests/FunctionalTests/FileUtil.cs

```diff
@@ -31,5 +33,20 @@ public static void NormalizeLineEndings(string source, string normalizedDest)
 
             File.WriteAllText(normalizedDest, contents);
         }
+
+        /// <summary>
+        /// Look for the given test file in the current directory as well as the "NetworkFiles" subdirectory
+        /// </summary>
+        /// <returns>The correct path to a valid test file</returns>
+        public static string TestFile(string source)
+        {
+            if (File.Exists(source))
+                return source;
+            else if (File.Exists(Path.Combine("NetworkFiles", source)))
+                return Path.Combine("NetworkFiles", source);
+            else
+                Debug.Fail("Could not find test assets for System.Net.NetworkInformation functional tests. Missing file: " + source);
```

Comment:
I think this should be some form of Assert.*, and not a call to the debug class.